Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebarItem.razor.cs

```diff
@@ -36,6 +36,12 @@ public partial class HxSidebarItem : IAsyncDisposable
 	/// </summary>
 	[Parameter] public bool ExpandOnMatch { get; set; } = true;
 
+	/// <summary>
+	/// Set to <c>false</c> if you don't want to highlight the item if one of the children URL matches.<br/>
+	/// Default is <c>true</c>.
+	/// </summary>
+	[Parameter] public bool HighlightOnActiveChild { get; set; } = true;
```

Comment:
In current state, I would probably go for `false` as the default.
Most of the sidebars I found are not highlighting the parent node et al.
A few do, but usually in different (less accented) style that the item itself.
The visual I like is GitHub:
![image](https://github.com/havit/Havit.Blazor/assets/12828888/c845f4c2-69ec-4f4e-8014-8cb24211c6da)

...consider making the visual style separately defined.
...consider "just bold" for the default parent highlighting (in such case I would vote for `true` as default value :-D)

Reply:
All done.